versions/schematic-2.md

```diff
@@ -0,0 +1,235 @@
+<a name="metadataOptionalMods"></a>OptionalMods | `NBT list` of `NBT string` | **Optional.** An array of mod ids which are needed to fully provide all features saved within this schematic. May be empty. This shall be saved as [NBT](#formFile) tag list of NBT tag string.
+
+##### Metadata Object Example:
+
+```NBT
+{
+    Name: "My Schematic",
+    Author: "Author Name",
+    Date: 1534701119L,
+    RequiredMods: [
+        "a_mod",
+        "another_mod"
+    ],
+    OptionalMods: [
+        "CraftBukkit"
+    ]
+}
+```
+
+#### <a name="paletteObject"></a>Palette Object
+
+An object which holds a mapping of a block state id to an index. The indices are given by the array index and won't be greater than [`2^(PaletteMax * 8) - 1`](#schematicPaletteMax).
+This object shall be a [NBT](#formFile) tag list of [NBT](#formFile) tag string.
+
+##### Block State
+
+A Block State is an instance of a block type with a set of extra data used to further define the material. The additional data varies by block type and a complete listing of vanilla types is available [here](http://minecraft.gamepedia.com/Block_states).
+
+The format of the Block State identifier is the [ID](#formId) of the block type and a set of comma-separated property `key=value` pairs surrounded by square brackets. If the block has no properties then they can be excluded. The block type [ID](#formId) is specified as a resource location.
+
+For example the air block has no properties so its representation would be just the block type [ID](#formId) `minecraft:air`. The planks block however has an enum property for the `variant` so its identification would be `minecraft:planks[variant=oak]`. Properties may be ordered with their keys in alphabetical ordering.
+
+##### Palette Object Example
+
+```NBT
+Palette: [
+    "minecraft:air",
+    "minecraft:planks[variant=oak]",
+    "a_mod:custom"
+]
+```
+
+#### <a name="tileEntityObject"></a>Tile entity object
+
+An object to specify a tile entity which is within the area. Tile entities are used by Minecraft to store additional data for a block (such as the lines of text on a sign or contents of a chest). The fields used to describe a tile entity vary for each type, however the structure will be the same as used by the [Minecraft Chunk Format](http://minecraft.gamepedia.com/Chunk_format#Block_entity_format).
+
+##### Fields
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="tileEntityProp"></a>{properties} | `<?>` | **Required**. Amount and format of fields depending on the tile entity, specified by Mojang. As listed within the [Minecraft Chunk Format](http://minecraft.gamepedia.com/Chunk_format#Block_entity_format).
+<a name="tileEntityPosX"></a>x | `NBT int` | **Required**. As specified by chunk format above, BUT this value must to be converted to relative coordinates of the area. Relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied).
```

Comment:
But i actually NEED to split them up, when applying them to a tile or use one of mojangs deserialization methods.
Because mojang specified them as splitted. Using any of the methods from mojang i'll need the format specified by mojang.

It's actually part of {properties}, i just separated it to mention, the values need to be converted relative.